backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/DocSubType.java

```diff
@@ -0,0 +1,133 @@
+package de.metas.document;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import de.metas.ad_reference.ReferenceId;
+import de.metas.util.lang.ReferenceListAwareEnum;
+import de.metas.util.lang.ReferenceListAwareEnums;
+import lombok.AllArgsConstructor;
+import lombok.NonNull;
+import org.compiere.model.X_C_DocType;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+@AllArgsConstructor
+public enum DocSubType implements ReferenceListAwareEnum
+{
+	ANY("ANY"),
+	NONE("NONE"),
```

Comment:
`ANY` and `NONE` are not doc sub types.
They are query filters or whatever else but not doc sub types.